flexmeasures/api/v3_0/tests/test_sensor_schedules.py

```diff
@@ -66,9 +66,10 @@ def test_trigger_and_get_schedule(
     )
 
     # check results are in the database
-    resolution = timedelta(minutes=15)
+    job.refresh()  # catch meta info that was added on this very instance
+    data_source_info = job.meta.get("data_source_info")
     scheduler_source = DataSource.query.filter_by(
-        name="Seita", type="scheduling script"
+        type="scheduling script", **data_source_info
     ).one_or_none()
     assert (
         scheduler_source is not None
```

Comment:
Also duplicate code, which could go to a handy util function, something like `def job_created_data_source(job) -> DataSource | None`.